content/security/authentication.md

```diff
@@ -805,7 +805,14 @@ export class JwtAuthGuard extends AuthGuard('jwt') {
   }
 }
 ```
+#### Extending guards with multiple strategies
```

Comment:
Do we need this heading? I believe this could fall into the existing `Extending guards` section.

Reply:
I agree but there should be something to separate the example from the one above.

Reply:
i removed the heading